lib/types/multipart.js

```diff
@@ -280,15 +278,16 @@ Multipart.prototype.end = function () {
       self._boy._done = true
       self._boy.emit('finish')
     })
-  } else if (this.parser.writable) { this.parser.end() }
```

Comment:
why is this condition not needed?

Reply:
I thought writable means it checks if it is a writable stream, but after checking the documentation, it seems to mean that it was not destroyed, reverted that